[2022-07-18 23:49:46,159] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2010-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:49:46,174] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2010-01-10T06:00:00+00:00 [queued]>
[2022-07-18 23:49:46,176] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:49:46,177] {taskinstance.py:1358} INFO - Starting attempt 2 of 2
[2022-07-18 23:49:46,179] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-07-18 23:49:46,197] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2010-01-10 06:00:00+00:00
[2022-07-18 23:49:46,206] {standard_task_runner.py:52} INFO - Started process 40333 to run task
[2022-07-18 23:49:46,211] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'local_to_gcs', 'local_to_gcs_task', 'scheduled__2010-01-10T06:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/local_to_gcs.py', '--cfg-path', '/tmp/tmp9zbnuyp8', '--error-file', '/tmp/tmph45a78l2']
[2022-07-18 23:49:46,213] {standard_task_runner.py:80} INFO - Job 79: Subtask local_to_gcs_task
[2022-07-18 23:49:46,281] {task_command.py:370} INFO - Running <TaskInstance: local_to_gcs.local_to_gcs_task scheduled__2010-01-10T06:00:00+00:00 [running]> on host 32d1f26135d5
[2022-07-18 23:49:46,360] {taskinstance.py:1572} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=local_to_gcs
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2010-01-10T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2010-01-10T06:00:00+00:00
[2022-07-18 23:49:53,471] {python.py:173} INFO - Done. Returned value was: None
[2022-07-18 23:49:53,535] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=local_to_gcs, task_id=local_to_gcs_task, execution_date=20100110T060000, start_date=20220718T234946, end_date=20220718T234953
[2022-07-18 23:49:53,623] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-18 23:49:53,698] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
